addons/account/models/account.py

```diff
@@ -538,6 +538,7 @@ class AccountGroup(models.Model):
     parent_path = fields.Char(index=True)
     name = fields.Char(required=True)
     code_prefix = fields.Char()
+    company_id = fields.Many2one('res.company', required=True, default=lambda self: self.env.company)
```

Comment:
why required? later in the security file you put `'|',('company_id','=',False)` which means it's not required 🤔